src/operator/api/v1alpha1/intents_types.go

```diff
@@ -152,6 +155,30 @@ func init() {
 	SchemeBuilder.Register(&ClientIntents{}, &ClientIntentsList{})
 }
 
+func (in *ClientIntentsList) FormatAsOtterizeIntents() ([]*intents.IntentInput, error) {
+	otterizeIntents := make([]*intents.IntentInput, 0)
+	for _, clientIntents := range in.Items {
+		for _, intent := range clientIntents.GetCallsList() {
+			b, err := json.Marshal(intent)
```

Comment:
This is a neat trick but it has the side effect of breaking silently if the fields change - some of them will just be ignored. This is losing a lot of the power of having these two separate APIs (GraphQL and Kubernetes) have generated statically typed code.

If you map the fields with code instead, the code will fail to compile if the structs change. Please do so - it will help those that come after you make changes easily and have confidence that they will work.

Reply:
Alright agree, will revert (it was like this originally). 
Just FYI I've aligned cloud intents to use the same format as the k8s intents. Should I keep the changes or revert them (I prefer the formats to be aligned but it doesn't matter if the conversion is code-based).

Reply:
Makes sense to have them aligned - they should represent the same object after all :)